test/test_jit.py

```diff
@@ -6251,12 +6251,60 @@ def forward(self, x):
                     x = torch.neg(x)
                 return x
 
+    def test_onnx_export_script_module_overload(self):
+        class ModuleToExport(torch.jit.ScriptModule):
```

Comment:
didn't you clobber the test above?

Reply:
The above test is verifying the `if` behavior. Did you mean this comment for the test below?

Reply:
No, I don't see how test_onnx_export_script_module_if actually tests anything anymore.  It doesn't instantiate a ModelToExport or run it through onnx, etc.

Reply:
Ah, right sorry. Saw it now.